### DISCUSSION TITLE
All dropdown menus incomplete and non-scrollable throughout the app, including demo site

### DISCUSSION THREAD

**Discussion by Talk1scheap** [firefly-iii/firefly-iii#9104] [Firefly III] [answered]
Issue Description:
Throughout the app, including the demo site, all dropdown menus appear to have the same problem: they only display a partial list of options without the ability to scroll and view the remaining hidden options. This issue was initially noticed with the destination account selection when creating a new expense record, but seems to affect all dropdown menus. Typing partial text in the input fields allows searching and finding the hidden options, but this workaround is not ideal for usability.

Expected Behavior:
All dropdown menus should either display all options or support scrolling to view all available options.

Steps to Reproduce:
1. Access the demo site
2. Open any interface in the app that contains a dropdown menu (e.g., creating a new expense record, selecting categories, etc.)
3. Click on any dropdown menu
4. Observe that only a partial list is shown and scrolling is not possible

Additional Information:
- This issue is reproducible on the demo site, indicating it's not specific to individual installations
- Browser: Latest version of Microsoft Edge
- Operating Systems: macOS (desktop) and Android (mobile)
- Issue occurs on both desktop and mobile platforms
- The problem appears to be consistent across all dropdown menus in the application

Suggested Fix:
Implement scrolling functionality for all dropdown menus throughout the app, or add pagination to display all options. This likely requires a global fix to the component or library used for dropdown menus.

Screenshots:
<img width="380" alt="image" src="https://github.com/user-attachments/assets/9120a0f4-ee80-477a-a4a9-98ca7a1e70f9">

**Comment by JC5** [comment, marked as answer]
Sorry, this will not change.

The fields you describe, that have this kind of dropdown, are free-format text fields. You can enter new or existing information, aided by the dropdown. It will always show a limited set of results when you open it with no text in the text field. The behavior you expect will not be implemented.

**Reply by Talk1scheap** [reply]
Thank you for the quick and clear explanation. I now understand these are free-format fields with suggestions, not traditional dropdowns. While initially confusing, I see the flexibility this offers. I appreciate your time and the work you put into this project.
